independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/project/quarkus/base/src/main/resources/META-INF/resources/index.tpl.qute.html

```diff
@@ -35,142 +56,241 @@
-    <div class="right-column">
-        <div class="right-section">
-            <h3>Application</h3>
-            <ul>
-                <li>GroupId: <code>{project.group-id}</code></li>
-                <li>ArtifactId: <code>{project.artifact-id}</code></li>
-                <li>Version: <code>{project.version}</code></li>
-                <li>Quarkus Version: <code>{quarkus.version}</code></li>
-            </ul>
-        </div>
-        <div class="right-section">
-            <h3>Do you like Quarkus?</h3>
-            <ul>
-                <li>Go give it a star on <a href="https://github.com/quarkusio/quarkus">GitHub</a>.</li>
-            </ul>
+</div>
+
+<div class="banner-wrapper">
+    <div class="grid">
+        <div class="banner">
+            <div class="grid">
+                <div class="callout">Congratulations!</div>
+                <div class="appdetails">
+                    <h5>Application</h5>
+                    <ul>
+                        <li>GroupId: <code>{project.group-id}</code></li>
+                        <li>ArtifactId: <code>{project.artifact-id}</code></li>
+                        <li>Version: <code>{project.version}</code></li>
+                        <li>Quarkus Version: <code>{quarkus.version}</code></li>
+                    </ul>
+                </div>
+            </div>
         </div>
-        {#if input.selected-extensions}
-        <div class="right-section">
-            <h3>Selected extensions guides</h3>
-            <ul>
-                {#for ext in input.selected-extensions}
-                {#if ext.guide}
-                <li title="{ext.description}"><a href="{ext.guide}" target="_blank">{ext.name} guide</a></li>
-                {/if}
-                {/for}
-            </ul>
+    </div>
+</div>
+
+<div class="content-wrapper">
+    <div class="grid">
+        <div class="left-column">
+            <h1>You just made a Quarkus application.</h1>
+            <h4>Try running Quarkus Dev and <a href="/q/dev/">Dev UI</a> to view your installed extensions.</h4>
+            <p>This page is served by Quarkus. The source is in <code>src/main/resources/{config.file-name}</code>.</p>
+            <p>Your static assets are located in <code>src/main/resources/META-INF/resources</code>.</p>
+            <p>Configure your application in <code>src/main/resources/{config.file-name}</code>.</p>
+            <p>Play with the provided code located in <code>{language.dir.code}</code>.</p>
+
+            <div class="provided-code">
+                <h3>These extensions were used in the creation of this project:</h3>
+<!-- Start restyle merged extensions content: {merged-content} --> 
+                <ul>
+                    <li>RESTEasy JAX-RS</li>
+                    <li>gRPC</li>
+                    <li>RESTEasy JAX_RS</li>
+                    <li>REASEasy Qute</li>
+                    <li>WebSockets</li>
+                </ul>
+<!-- End restyle merged extensions content: {merged-content} --> 
```

Comment:
This was supposed to be replaced with the actual list of extensions, but apparently that's not happening

Reply:
The is something we discussed with Erin that will require a change.

@ebullient  - How do we update the system to generate the the list of extensions?

Reply:
You need this code from the previous version of the file: 
```
{merged-content}
```

Reply:
ok... y'all will then format the content into the UL?

Reply:
The `{merged-content}` doesn't seem to add the list of extensions here, but maybe I'm missing something. @ebullient can I ask you to give it a try?

Reply:
OK, so things are not perfect here but I'd like to discuss them with Andy when he's back. In the meantime, I will merge this.